### ISSUE TITLE
Problems with writeManifest for RODBC due to latest having R version 4 dependency

### ISSUE THREAD

**Issue by hollywoof** [rstudio/rsconnect#431]
I'm encountering an issue with writing manifest files for RStudio Connect at present.

I think this is due to the latest RODBC requiring R version 4 - which means that the current version is unavailable in my current R version (3.6.3).

I think the `writeManifest` call is having problems due to the `available.packages` call on https://github.com/rstudio/rsconnect/blob/40c162bf3c4cd9b10deed82e1df9ec1b0c107c1f/R/dependencies.R#L83

It causes a warning "Unable to find repository URL for package RODBC" and causes the manifest to output as:
```
    "RODBC": {
      "Source": "CRAN",
      "Repository": null,
      "GithubRepo": null,
````

which then causes the deployment to fail in RSConnect (via git deployment)

My workaround for this at present is to set `options(available_packages_filters = character())` before calling `writeManifest` - but it feels like `writeManifest` should somehow handle this problem internally - maybe by setting the `filters` in the call to `available.packages` or maybe by searching in 
the archives for each repo.

**Comment by hollywoof** [comment]
Apologies for the double commit - http timeout issue (dodgy broadband)

**Comment by slodge** [comment]
This was merged a few months ago - closing

**Comment by hadley** [comment]
Looks like this was fixed in #432.
